lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -440,6 +441,10 @@ def write(chunk)
             # determining whether we should parse the log or message field.
             timestamp = record.key?('time') ? record['time'] : nil
             record.delete('time')
+            if timestamp.nil?
+              timestamp = record.key?('timestamp') ? record['timestamp'] : nil
+              record.delete('timestamp')
```

Comment:
Careful, this will break the code that restores the `'time'` field in line 462 (which you'd probably want to do with the `'timestamp'` field as well). I'd suggest having two separate variables for these, `time_value` and `timestamp_value`.